File: app/src/main/java/com/example/miwok/MainActivity.java
package com.example.miwok;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        intents();
    }


    public void intents(){
        TextView colors = findViewById(R.id.colors);
        colors.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Colors.class);
                startActivity(intent);
            }
        });

        TextView numbers = findViewById(R.id.numbers);
        numbers.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Numbers.class);
                startActivity(intent);
            }
        });

        TextView phrases = findViewById(R.id.phrases);
        phrases.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Phrases.class);
                startActivity(intent);
            }
        });

        TextView family = findViewById(R.id.family);
        family.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, Family.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/miwok/MainActivity.kt
package com.example.miwok

import android.content.Intent
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<TextView>(R.id.colors).setOnClickListener {
            startActivity(Intent(this, Colors::class.java))
        }

        findViewById<TextView>(R.id.numbers).setOnClickListener {
            startActivity(Intent(this, Numbers::class.java))
        }

        findViewById<TextView>(R.id.phrases).setOnClickListener {
            startActivity(Intent(this, Phrases::class.java))
        }

        findViewById<TextView>(R.id.family).setOnClickListener {
            startActivity(Intent(this, Family::class.java))
        }
    }
}


